src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -494,10 +498,13 @@ private void FinishAttachToMicrobe()
         }
 
         // Physics
-        // TODO: shouldn't we also add the mass to the colony master?
         ParentMicrobe!.Mass += Definition.Mass;
 
-        MakeCollisionShapes(ParentMicrobe.Colony?.Master ?? ParentMicrobe);
+        // Add the mass of the organelles to the colony master
+        if (ParentMicrobe.Colony != null && ParentMicrobe != ParentMicrobe.Colony.Master)
+            ParentMicrobe.Colony.Master.Mass += Definition.Mass;
```

Comment:
I'm a bit suspicious that this line might cause duplicate mass to be added to the colony leader. As If we load a save there's we add the masses of the individual cells to the leader, and then call FinishAttachToMicrobe for all organelles, which adds the mass again. So I think there is a bug maybe here.
And I think maybe the bug would go entirely away if this wouldn't add any mass to the colony master.

Reply:
cant colony members grow themselves organelles in colony?

Reply:
at least that was what i was thinking

Reply:
They cannot. That is entirely prevented. Even in my multicellular prototype there's no organelle division. Only fully formed cells are added once there are enough resources.

Then could you comment out that bit of code and put like a TODO:
`TODO: if organelles can grow while cells are in a colony this will be needed (when enabled something needs to be modified to ensure no duplicate mass is added with loading a save)`

Reply:
ill modify it to only wotk when not loading a save and then comment

Reply:
It seems you commented it out already. Are you going to not do this:

> ill modify it to only wotk when not loading a save and then comment

Also I'd like the warning bit that it will cause a bug if it isn't adjusted in the future, as I'm quite sure that whoever uncomments the code in teh future won't know that we had this conversation.

Reply:
i did? i put a !IsLoadedFromSave in the if

Reply:
Ah, sorry, I didn't see that. I think it's fine then once the comment format is fixed so that the CI is happy.